static/src/stylesheets/module/identity/_wizard.scss

```diff
@@ -1,12 +1,41 @@
 .manage-account-wizard {
     @include clearfix();
+    position: relative;
+    transition: min-height .2s;
+    will-change: min-height;
     &--completed {
         .manage-account-wizard__controls {
             display: none;
         }
     }
 }
 
+.manage-account-wizard__step {
+    top: 0;
+    left: 0;
+    right: 0;
+    will-change: transform;
+    &--hidden {
+        display: none;
+    }
+    &--out {
```

Comment:
if you can be explicit and avoid &--hidden type selectors, it make it much easier to grep for a classname if you need to find it